assets/src/components/Sidebar/NewThemeSettings.js

```diff
@@ -61,10 +61,6 @@ export const applyChangesToDom = (theme, initialVars) => {
     document.documentElement.style.setProperty(name, value);
   });
 
-  if (initialVars.length === 0) {
```

Comment:
I'm not sure why I did this earlier, but it broke switching back to the default theme, so probably a mistake.